browser/brave_stats/brave_stats_updater.cc

```diff
@@ -265,6 +271,11 @@ bool BraveStatsUpdater::IsReferralInitialized() {
 #endif
 }
 
+bool BraveStatsUpdater::IsAdsEnabled() {
+  return ProfileManager::GetPrimaryUserProfile()->GetPrefs()->GetBoolean(
```

Comment:
Users could have ads enabled on another profile but disabled on their primary profile. Is this intentional?

Reply:
Yeah for now it's fine. It's how we calculated it for the `adsEnabled` param which has given us good results. Stats is also just a heuristic for these things.